zypp/target/rpm/librpmDb.cc

```diff
@@ -125,6 +125,10 @@ bool librpmDb::globalInit()
   initialized = true; // Necessary to be able to use exand().
   _rpmDefaultDbPath = expand( "%{_dbpath}" );
 
+  if ( _rpmDefaultDbPath.empty() ) {
+    _rpmDefaultDbPath = "/usr/lib/sysimage/";
```

Comment:
should we make that a  constexpr std::string_view somewhere ?

Reply:
IMO not needed yet. This is the only 'volatile' location. In case the distro changes the default again, it would be changed here. 
Down in suggestedDbPath the path occurs again, but in a list of known legacy paths where a DB could exist. But there the value is const and only more locations could be added.